packages/storybook/src/react-components/heading/Heading.mdx

```diff
@@ -0,0 +1,37 @@
+import { Canvas, Controls, Description, Markdown, Meta } from "@storybook/blocks";
+import markdown from "@utrecht/heading-css/README.md?raw";
+import * as HeadingStories from "./Heading.stories";
+import { CitationDocumentation } from "../../utils/CitationDocumentation";
+
+<Meta of={HeadingStories} />
+
+# Lux Heading
+
+Lux Heading is gebasseerd op Utrecht Heading.
```

Comment:
Dit blokje mag weg. Het is niet van belang voor de gebruiker om te weten welk component onderwater gebruikt wordt

Reply:
Alleen deze nog. Daarna zijn we klaar